### DISCUSSION TITLE
Healthcare? Any users out there?

### DISCUSSION THREAD

**Discussion by mdav43** [apache/superset#18043] [General]
Hi - wanted to reach-out to the community to understand any users that have rolled out production to healthcare organisation (hospital, research group etc)?

I'm interested in developing a business case for some units in Australia and want to have an opportunity to talk to people that have already executed a rollout.

**Comment by mayasirode** [comment]
Hey Marcus,

even though it's been 2 years, I wonder what happened to your project. I'm currently testing superset for reporting in a hospital and am therefore interested in your experiences. Best regards Simon

**Reply by mdav43** [reply]
Hey I've moved on from the hospitals and now in the financial sector. Similar problems etc.. 

I've evolved my reporting to strike a balance. It's important for me to understand what end users are actually after. I've found some require superset - to go and explore data sets and get in-depth reporting (PULL REPORTING) - but the majority just want an email or summary report (PUSH REPORTING) so they can keep an eye on MOM, QoQ and YoY movements/KPIs. 

I'd still recommend Superset and still use it to this day.

**Reply by mdav43** [reply]
For the push reporting, I like either a qmd script or  python notebook (with some paper mill automation) does the trick well.

A lot of my decisions have been based on the huge restrictions in the environments I have had to build things. I have even setup scripts to push standalone html files to Sharepoint that are dashboards. 

Some interesting tools I have come across: 
[Papermill](https://github.com/nteract/papermill)
[Evidence.dev](https://evidence.dev)

**Comment by mayasirode** [comment]
Hey Marcus,

Thank you very much for your Response!! Thats very inspiring. I totaly
agree to your opinion about the reporting (pull vs push).

My project is on a god way, we are installing apache superset in the
Networks Environment of the hospital.

Best regards

Simon



Marcus ***@***.***> schrieb am Di., 27. Feb. 2024, 12:03:
